modules/nf-core/mcquant/main.nf

```diff
@@ -0,0 +1,36 @@
+process MCQUANT {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "labsyspharm/quantification:1.5.4"
+
+    input:
+    tuple val(meta), path(image)
+    tuple val(meta2), path(mask)
+    tuple val(meta3), path(markerfile)
+
+    output:
+    tuple val(meta), path("*.csv"), emit: csv
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    // TODO: Replace fixed version number by mcquant command. PR to Mcquant needed!
+    """
+    python /app/CommandSingleCellExtraction.py \
+        --masks $mask \
+        --image $image \
+        --output . \
```

Comment:
Is the output file name fixed? If so can you change it to use the prefix? (mainly to prevent file name collisions downstream)

Reply:
The output file naming is a bit complex at the moment. Capturing any csv should be fine for now, as only one will be generated or would be different with different input filenames. We might open a PR on the MCQUant repo to make naming explicit, but for now I would like to keep it like this.